core/domain/config_services_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+import collections
+
+from core.domain import config_domain
+from core.domain import config_services
+from core.tests import test_utils
+
+
+class ConfigServicesDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ConfigServicesDomainUnitTests, self).setUp()
+        ConfigObj = collections.namedtuple(
+            'ConfigObj', 'name new_config_value')
+        self.committer_id = u'configServicesCommitterID'
+        self.except_str = 'No config property with name.+'
+        self.config_obj = ConfigObj(
+            name='propertyName', new_config_value='propertyValue')
+        self.invalid_property_name = u'_no_such_property_name'
+        self.property_dummy_value = u'TEST_VALUE'
+        self.schema_unicode = 'unicode'
+        self.cfg_schemas = config_domain.Registry.get_config_property_schemas()
+        # Get first property name, with unicode as schema type.
+        self.property_name = next(
+            x for x in self.cfg_schemas if
+            self.cfg_schemas[x]['schema']['type'] == self.schema_unicode)
+
+    def test_set_property(self):
+        with self.assertRaisesRegexp(Exception, self.except_str):
```

Comment:
Let's not use the variable here. We should try to keep it consistent with the other tests. I suggest we go with something like:
```
with self.assertRaisesRegexp(Exception, (
       'No config property with name.+')):
```
Ditto below.

Reply:
This is so weird :)
Lint fails me on indentation like you provided.
If i remove one tab then it passes BUT then it looks ugly & I get a warning from IDE that it is not according to PEP8. There is something weird going on.
I will upload how Lint allows for now.

Reply:
Just to confirm, there is only one tab required here, something like [this example](https://github.com/oppia/oppia/blob/develop/core/domain/classifier_domain_test.py#L112).